components/ui/authbutton.tsx

```diff
@@ -14,25 +15,40 @@ export default async function AuthButton() {
   };
 
   return (
-    <div className="hidden md:flex">
+    <div className="hidden flex-row items-center space-x-1 md:flex">
       {error || !data?.user ? (
         <>
-          <Link
-            href="/signin"
-            className="mx-4 my-2 inline-flex items-center justify-center rounded-sm border border-transparent bg-primary-500 px-4 py-2 font-medium text-white-main transition duration-150 ease-in-out hover:bg-primary-600"
+          <svg
```

Comment:
- Remove this svg and use CircleUserRound icon since we have lucide-react library 
- make sure to wrap the CircleUserRound icon in the <Link> component
eg. 
```js
 <Link href={"/signin"} className="flex items-center space-x-1 transition duration-150 ease-in-out hover:text-gray-600">
<CircleUserRound className="h-4 w-4" />
<span>Sign in</span>
</Link>
```

Reply:
ooh okay

Reply:
i din know we were using that library!

Reply:
done